test/support.ts

```diff
@@ -153,14 +153,15 @@ export async function prepareTransactionTest(sequelize: Sequelize) {
 }
 
 export function createSequelizeInstance(options: Options = {}) {
-  options.dialect = getTestDialect();
+  options.dialect = getTestDialect(); // only returns translated dialectName "mysql8"->"mysql"
```

Comment:
This comment belongs in the JSDoc of getTestDialect instead

```suggestion
  options.dialect = getTestDialect();
```